db/migrate/20230709181609_drop_archived_container_projects.rb

```diff
@@ -0,0 +1,15 @@
+class DropArchivedContainerProjects < ActiveRecord::Migration[6.1]
+  class ContainerGroup < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ContainerProject < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
```

Comment:
Minor but this class isn't used here and could be moved into the spec file since it is only being used for tests